fawltydeps/settings.py

```diff
@@ -327,11 +327,11 @@ def populate_parser_options(parser: argparse._ActionsContainer) -> None:
         ),
     )
     parser.add_argument(
-        "--venv",
+        "--pyenv",
         type=Path,
-        metavar="VENV_DIR",
+        metavar="DIR",
```

Comment:
Why `DIR` instead of `PYENV_DIR`?

Reply:
🤷🏻‍♂️ I'm good with either option.

```suggestion
        metavar="PYENV_DIR",
```

Reply:
Let's go with the `PYENV_DIR` then :D